definitions/npm/jest_v24.x.x/flow_v0.61.x-v0.103.x/jest_v24.x.x.js

```diff
@@ -590,7 +590,6 @@ type SnapshotDiffType = {
     |},
     testName?: string
   ): void,
-  ...
```

Comment:
This is incorrect in older flow versions and was causing an error, so I removed it. Of course I left it in the libdef for the newer versions.